examples/pennylane_run_advanced_usage.py

```diff
@@ -142,3 +142,38 @@ def circuit3(param, fixed=None):
 # --> 136         value = self.kwarg_values[self.name][self.idx] * self.mult
 #     137         return value
 # TypeError: unsupported operand type(s) for *: 'NoneType' and 'int'
+#
+# QNodes from different interfaces on one Device
+# -----------------------------------------------
+#
+# In addition to allowing multiple quantum nodes on one device,
+# PennyLane also allows provides the flexibility for quantum nodes to have multiple interfaces.
+# Let's look at the following simple example:
+
+dev1 = qml.device('default.qubit', wires=1)
+def circuit(phi):
+    qml.RX(phi, wires=0)
+    return qml.expval(qml.PauliZ(0))
+
+##############################################################################
+# Now, we construct multiple QNodes on the same device and change the interface of one of them 
+# from NumPy to PyTorch: 
+
+qnode1 = qml.QNode(circuit, dev1)
+qnode2 = qml.QNode(circuit, dev1)
+qnode1_torch = qnode1.to_torch()
+
+##############################################################################
+# Let's define the cost function that accepts a QNode as an argument.
+   
+def cost(qnode, phi):
+    return qnode(phi)
+
+##############################################################################
+# Now we can call the cost function as follows:
+
+print(cost(qnode1, np.pi))
```

Comment:
These print statements do not render anything when I build locally. Do you see them when you build?

Reply:
Yes, I do.

Reply:
![advanced](https://user-images.githubusercontent.com/44415402/65608619-8150b680-df7c-11e9-8882-c4356967bd5f.png)

Reply:
There is some issue with the 'shots' argument for a device which might be why you don't see the whole script run. Let me ask Josh

Reply:
The code structure around the 'shots' argument of a device has been updated and we have to reinstall pennylane

Reply:
Yep, there are two issues here, both related to #317:

1. Since `default.qubit` and `default.gaussian` were modified, likely that PL needs to be re-installed so that the 'changes' are registered by setuptools (`pip install -e .`).

2. The various plugins (SF, Forest, PQ) need to be updated to match the api change (qiskit has already been fixed in a PR).

Reply:
got it thanks